test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -1215,8 +1216,10 @@ class test_ucp_sockaddr_wireup_fail : public test_ucp_sockaddr_wireup {
 
         if (m_num_fail_injections == 0) {
             EXPECT_EQ(0, m_err_count);
-            UCS_TEST_SKIP_R("failure injection was not done, but endpoint was"
-                            " connected");
+            UCS_TEST_MESSAGE <<
+                    "failure injection was not done, but endpoint connected";
```

Comment:
just "failure injection was not done"
("but" does not fit here)

Reply:
done